docs/methods.rst

```diff
@@ -1174,7 +1174,7 @@ Additional information and recommendations:
    parameters discussed earlier for descent are useful for setting up
    the descent engine to do this averaging on its own.
 
-Stochastic Reconfiguration
+Stochastic Reconfiguration with Gradient Descent
```

Comment:
Can we have `Conjugate Gradient` instead of `Gradient Descent`?

Reply:
yep thats what I meant to put. Working too fast. Will fix